pkg/hacl/Makefile

```diff
@@ -0,0 +1,13 @@
+PKG_NAME=hacl
+PKG_URL=https://github.com/mitls/hacl-c
+PKG_VERSION=aac05f5094fc92569169d5a2af54c12387160634
+PKG_LICENSE=MIT
+
+.PHONY: all
+
+all: git-download
+	@cp $(RIOTBASE)/pkg/hacl/src/* $(PKG_BUILDDIR)
+	@cp Makefile.hacl $(PKG_BUILDDIR)/Makefile
+	"$(MAKE)" -C $(PKG_BUILDDIR)
```

Comment:
You can replace line 10 and 11 here by:
`"$(MAKE)" -C $(PKG_BUILDDIR) -f $(CURDIR)/Makefile.$(PKG_NAME)`
As done in the cayenne-lpp  package, or with
`"$(MAKE)" -C $(PKG_BUILDDIR) -f $(CURDIR)/Makefile.hacl`
as for example done in the cn-cbor and heatshrink packages.